repositories/src/main/scala/cromwell/pipeline/datastorage/dto/auth/PasswordProblemsResponse.scala

```diff
@@ -0,0 +1,3 @@
+package cromwell.pipeline.datastorage.dto.auth
+
+final case class PasswordProblemsResponse(value: String, errors: List[Map[String, String]])
```

Comment:
It's a bad naming. 
`Problems` is not appropriate words. Please come up with another naming. 

Also as far as I can see, this case class intended to be a container for all request validation errors. It could be this way, but in that case case class **must** be type based, not string based. 
I find it useless to have `List[Map[String, String]]`. It says nothing about about errors at the type level.